core/src/main/resources/jenkins/install/pluginSetupWizard.properties

```diff
@@ -56,3 +58,20 @@ installWizard_pluginInstallFailure_message=Some plugins failed to install proper
 with the failed plugins
 installWizard_continue=Continue
 installWizard_retry=Retry
+installWizard_upgradePanel_title=Upgrade
+installWizard_upgradePanel_banner=Welcome to Jenkins {0}!
+installWizard_upgradePanel_message=Jenkins {0} includes some great new features that we think you'll love, install these additional plugins to take advantage of them!
+installWizard_upgradePanel_recommendedActionTitle=Install Suggested Plugins
+installWizard_upgradePanel_recommendedActionDetails=Install the suggested plugins
+installWizard_upgradePanel_skipRecommendedPlugins=No thanks
+installWizard_upgradeComplete_title=Upgrade
+installWizard_pluginsInstalled_banner=Welcome to Jenkins {0}!
+installWizard_upgradeComplete_message=Congratulations! You have upgraded to Jenkins {0}.</p><p>To learn more about the great new features, visit <a target="_blank" href="https://jenkins.io/2.0/">the Jenkins website</a>!
+installWizard_upgradeSkipped_title=Upgrade
+installWizard_upgradeSkipped_banner=Features Not Installed
+installWizard_upgradeSkipped_message=<div class="alert alert-warning fade in">No new plugins will be installed</div> \
+<p style="padding: 0 4px">You can also install new features from the Plugin Manager, if you change your mind.</p> \
+<p style="padding: 0 4px">Visit the <a target="_blank" href="https://jenkins.io/2.0/">Jenkins 2 website</a> \
```

Comment:
> Visit the Jenkins 2 website to learn how these new features can improve your Jenkins experience.

This sentence strikes me as awkward but I'm having difficulty explaining what it is exactly. If this were in documentation or a page on the site, I generally try to end the "call to action" with the link. I think that kind of sentence structure is more likely to get somebody to click, since they're not visually back-tracking to click the right link (perhaps I spend too much time thinking about these kinds of things :smile: )

I suggest:

> Learn how these new features can improve your Jenkins experience on the **Jenkins 2 homepage**